website/docs/guides/development-helm-chart.md

```diff
@@ -115,12 +115,19 @@ Install the chart in Kubernetes:
 scripts/helm/$ helm install ort-server ./ort-server --namespace ort-server --create-namespace
 ```
 
-Switch to the newly created namespace with https://github.com/ahmetb/kubectx/blob/master/kubens[KubeNS]:
+Switch to the newly created namespace with [KubeNS](https://github.com/ahmetb/kubectx):
```

Comment:
Commit message nit: The body still has a single bullet item. Please just use

    Relates to #1631.

instead.

Reply:
Similar for the other commits, BTW: Use simple present "Relates to" instead of "related".